flake.nix

```diff
@@ -3,6 +3,7 @@
     flakeModule = ./nix/flake-module.nix;
 
     lib = ./nix/lib.nix;
+    evalModules = import ./nix/eval-modules.nix;
```

Comment:
I thought about including this in lib, since lib.nix is a function needing "lib", I was not sure..